index.js

```diff
@@ -5,18 +5,26 @@ module.exports = (context = {}) => {
 	const { useConfigFile = true, configFilePath, plugins = [] } = context
 
 	if (useConfigFile === false) {
-		return ({ content }) => Promise.resolve(process(plugins, content))
+		return ({ content, filename }) => Promise.resolve(process(plugins, content, filename))
 	} else {
 		const configPromise = postcssLoadConfig(context, configFilePath)
 
-		return ({ content }) => configPromise.then(
-			({ plugins }) => process(plugins, content)
+		return ({ content, filename }) => configPromise.then(
+			({ plugins }) => process(plugins, content, filename)
 		)
 	}
 }
 
-function process(plugins, css) {
+function process(plugins, css, filename) {
 	return postcss(plugins)
-		.process(css)
-		.then(code => ({ code }))
+		.process(css, {
+			from: filename,
```

Comment:
So PostCSS is cool with a `from` option property whose value is `undefined`?

Reply:
`undefined`? Svelte passes the `filename` into the callback which then gets passed to the `process` function and into postcss `from`. I tested it and it works fine on my system... am I missing something obvious?

Note: In the latest commit I fixed the name from `fileName` to `filename`, do you have that commit locally?